src/ot/Transformation.js

```diff
@@ -1,4 +1,36 @@
 var { Operation, PathUtils, Mark } = require('slate');
+
+const insertNodeTransformHelper = (op1, insertNodeOp) => {
+  const newPath = PathUtils.transform(op1.get('path'), insertNodeOp).get(0);
+  if (newPath) {
+    return Operation.create({
+      ...op1.toJSON(),
+      path: newPath,
+    });
+  }
+
+  return op1;
+};
+
+const removeNodeTransformHelper = (op1, removeNodeOp) => {
+  const newPath = PathUtils.transform(op1.get('path'), removeNodeOp).get(0);
+  if (newPath) {
+    return Operation.create({
+      ...op1.toJSON(),
+      path: newPath,
+    });
+  }
+
+  const nodeIsBeingRemoved =
```

Comment:
I think the PathUtils.transform already takes care of this step. So we can merge insertNodeTransformHelper and removeNodeTransformHelper together.

Reply:
I'll take a look at this one, not sure if we can merge in